go/hyperkit.go

```diff
@@ -305,14 +305,16 @@ func (h *HyperKit) execute(cmdline string) error {
 			config.Path = filepath.Clean(filepath.Join(h.StateDir, fmt.Sprintf("disk%02d.img", idx)))
 			h.Disks[idx] = config
 		}
-		if _, err = os.Stat(config.Path); os.IsNotExist(err) {
-			if config.Size != 0 {
-				err = CreateDiskImage(config.Path, config.Size)
-				if err != nil {
-					return err
+		if !strings.HasPrefix(config.Path, "file://") {
```

Comment:
I don't understand this change nor the commit message

Reply:
`qcow-tool` wants the file to be specified as `"file:///this/is/my/image.qcow?sync=1&buffered=1"`.  Currently Docker for Mac passes this as "the path" since hyperkit.go offers no support for qcow, and the string must be passed as is to the driver.

So, as a transitional step until proper qcow support is implemented in hyperkit.go, we just need the existing check to be ignored for these files, as docker for mac already took care of this.

This will be removed, of course.

Reply:
something with this sort of detail would be good to have in the commit message.

Reply:
Indeed.